Plugin: 原神cos

**插件名称：**

原神cos

**插件功能：**

获取米游社cos图片

**PyPI 项目名：**

nonebot-plugin-genshin-cos

**插件 import 包名：**

nonebot_plugin_genshin_cos

**插件项目仓库/主页链接：**

https://github.com/Cvandia/nonebot_plugin_genshin_cos

**标签：**

[{"label":"t:原神","color":"#f55400"},{"label":"t:cos","color":"#00d0f5"},{"label":"t:coser","color":"#eb1dd3"}]

# 📃 商店发布检查结果

> Plugin: 原神cos

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-genshin-cos 的信息如下：
    name         : nonebot-plugin-genshin-cos 
     version      : 0.1.1                      
     description  : 米游社原神cos图获取                
    
    dependencies
     - aiofiles &gt;=0.7.0
     - httpx &gt;=0.19.0
     - nonebot-adapter-onebot &gt;=2.0.0-beta.1
     - nonebot-plugin-apscheduler &gt;=0.1.3
     - nonebot2 &gt;=2.0.0-beta.1
     - pathlib &gt;=1.0.1
     - pydantic &gt;=1.10.2
     - ujson &gt;=5.5.0
插件 nonebot-plugin-genshin-cos 依赖的插件如下：
    nonebot_plugin_apscheduler
插件 nonebot_plugin_genshin_cos 加载出错：
    03-28 12:37:02 [SUCCESS] nonebot | NoneBot is initializing...
    03-28 12:37:02 [INFO] nonebot | Current Env: prod
    03-28 12:37:02 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_genshin_cos&quot;
    Traceback (most recent call last):
      File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
        valid = load_plugin(&quot;nonebot_plugin_genshin_cos&quot;)
      File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
        return manager.load_plugin(module_path)
    &gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
        module = importlib.import_module(name)
      File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
      File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
      File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
      File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
      File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
        super().exec_module(module)
      File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
      File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
      File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_genshin_cos/__init__.py&quot;, line 4, in &lt;module&gt;
        from .utils import get_cos,WriteError
      File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_genshin_cos/utils.py&quot;, line 1, in &lt;module&gt;
        import requests
    ModuleNotFoundError: No module named &#x27;requests&#x27;
    03-28 12:37:02 [SUCCESS] nonebot | Succeeded to import &quot;nonebot_plugin_apscheduler&quot;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: t:原神-#f55400, t:cos-#00d0f5, t:coser-#eb1dd3。</li><li>✅ 项目 <a href="https://github.com/Cvandia/nonebot_plugin_genshin_cos">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-genshin-cos/">nonebot-plugin-genshin-cos</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -5022,5 +5022,28 @@
     "homepage": "https://github.com/NumberSir/nonebot-plugin-questionmark",
     "tags": [],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_genshin_cos",
+    "project_link": "nonebot-plugin-genshin-cos",
+    "name": "原神cos",
+    "desc": "获取米游社cos图片",
+    "author": "Cvandia",
+    "homepage": "https://github.com/Cvandia/nonebot_plugin_genshin_cos",
+    "tags": [
+      {
+        "label": "t:原神",
+        "color": "#f55400"
+      },
+      {
+        "label": "t:cos",
+        "color": "#00d0f5"
+      },
+      {
+        "label": "t:coser",
+        "color": "#eb1dd3"
+      }
+    ],
+    "is_official": false
   }
 ]

